comms/lighthouse/BUILD.bazel

```diff
@@ -110,7 +110,10 @@ ts_library(
         ":lighthouse",
         "//commons/servers",
         "//commons/utils",
+        "@npm//@types/express",
         "@npm//@types/jasmine",
+        "@npm//isomorphic-fetch",
```

Comment:
I would recommend using [cross-fetch](https://www.npmjs.com/package/cross-fetch) instead. On the `catalyst-client` we had some problems with `isomorphic-fetch`.

Reply:
This is just for testing so it shouldn't matter that much